manualStacks/dns/README.md

```diff
@@ -33,33 +33,33 @@ Replace `<environment>` with `dev`, `build`, `staging`, `integration`, `producti
 
 ### Creating a New Stack
 
-Set your AWS profile to the correct environment, the run:
+Set your AWS profile to the correct environment, run `export ENVIRONMENT=<environment>` then run:
```

Comment:
You also need to log in to the correct AWS profile for running the command, is this confusing that as well?

Reply:
I intended it read as a list of instructions. I'd be happy to add an oxford comma or take other suggestions

Reply:
yeah could you add this pls 
```suggestion
Set your AWS profile to the correct environment, then run `export ENVIRONMENT=<environment>`, then run:
```